src/Pages/URLShortener/URLShortener.js

```diff
@@ -425,7 +425,15 @@ export default function URLShortenerPage(props) {
                         className={`${className}`}
                         key={title}
                       >
-                        <button onClick={() => handleSortUrls(columnName)}>{title}</button>
+                        <div className="flex items-center justify-center">
+                          <button onClick={() => handleSortUrls(columnName)}>{title}</button>
+                          <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth="1.5" stroke="currentColor" className={`w-5 h-5 ${(currentSortOrder === 'ASC' && currentSortColumn === columnName) ? `` : 'hidden'}`}>
```

Comment:
can this class be a function call instead? would be cleaner like

`className={getSortArrowClass(columnName, 'ASC')}`

the function would basically do the same logic, taking a param for `columnName` and `sortOrder` returning a blank string or `'hidden'`.

Reply:
> @evanugarte resolved